src/modules/bids/cqrs/commands/handlers/place-bid-on-item.command.handler.ts

```diff
@@ -0,0 +1,60 @@
+import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
+import { PlaceBidOnItemCommand } from '../implements/place-bid-on-item.command';
+import { Inject, NotFoundException } from '@nestjs/common';
+import { BidRepository } from 'src/modules/bids/repository/bid.repository';
+import { UserRepository } from 'src/modules/users/repository/user.repository';
+import { ItemRepository } from 'src/modules/items/repository/item.repository';
+import { BidDomainService } from 'src/modules/bids/domain/bid-domain.service';
+import { DataSource } from 'typeorm';
+
+@CommandHandler(PlaceBidOnItemCommand)
+export class PlaceBidOnItemCommandHandler
+  implements ICommandHandler<PlaceBidOnItemCommand>
+{
+  constructor(
+    @Inject(BidRepository)
+    private readonly bidRepository: BidRepository,
+    @Inject(UserRepository)
+    private readonly userRepository: UserRepository,
+    @Inject(ItemRepository)
+    private readonly itemRepository: ItemRepository,
```

Comment:
Về nguyên tắc thì không cần mấy cái annotation này

```suggestion
    private readonly bidRepository: BidRepository,
    private readonly userRepository: UserRepository,
    private readonly itemRepository: ItemRepository,
```